ere-guests/stateless-validator/ethrex/risc0/src/main.rs

```diff
@@ -60,5 +56,5 @@ fn commit_output(block_hash: [u8; 32], parent_hash: [u8; 32], is_valid: bool) {
     let public_inputs = (block_hash, parent_hash, is_valid);
     let public_inputs_hash: [u8; 32] =
         Sha256::digest(bincode::serialize(&public_inputs).unwrap()).into();
-    env::commit_slice(&public_inputs_hash);
+    P::write_whole_output(&public_inputs_hash);
```

Comment:
Could we use the same `Risc0Plaform<..>` style as with the Reth guest to avoid the tuple serialization and hashing on the guest program? (same with SP1).